packages/material-ui/src/test-utils/describeCustomVariantsConformance.js

```diff
@@ -0,0 +1,100 @@
+import { expect } from 'chai';
+import * as React from 'react';
+import { createClientRender, screen } from 'test/utils/createClientRender';
+import {
+  createMuiTheme,
+  ThemeProvider,
+} from '../styles';
+
+/**
+ * Tests whether the custom variants defined in the theme work as expected.
+ *
+ * @param {Component} - the component with it's minimal required props
+ * @param {string} name
+ *
+ */
+export default function describeCustomVariantsConformance(Component, name) {
+  describe('Material-UI custom variants', () => {
+    const isJSDOM = navigator.userAgent === 'node.js';
+
+    const render = createClientRender();
+
+    const WrappedComponent = ({ theme, ...props }) => {
+      return <ThemeProvider theme={theme}><Component data-testid="component" {...props}>Content</Component></ThemeProvider>
+    }
+
+    it('should map the variant classkey to the component', () => {
```

Comment:
Seems like you either have to name the test function (otherwise `this` refers to `describeCustomVariantsConformance` untranspiled) or you can move the skip up to the describe block (and name the describe callback).

```suggestion
    it('should map the variant classkey to the component', function test() {
```

Reply:
Yep, used named functions in the `it`, will leave it there 👍